.ci/docker/build.sh

```diff
@@ -208,20 +208,6 @@ case "$image" in
     CONDA_CMAKE=yes
     TRITON=yes
     ;;
-  pytorch-linux-focal-cuda12.4-cudnn9-py3-gcc9)
```

Comment:
These are 3 duplicated entries of `pytorch-linux-focal-cuda12.4-cudnn9-py3-gcc9` in the docker build script